tests/typ/math/cancel.typ

```diff
@@ -0,0 +1,38 @@
+// Tests the cancel() function.
+
+---
+// Inline
+$a + 5 + cancel(x) + b - cancel(x)$
+
+$c + (a dot.c cancel(b dot.c c))/(cancel(b dot.c c))$
+
+---
+// Display
+#set page(width: auto)
+$ a + b + cancel(b + c) - cancel(b) - cancel(c) - 5 + cancel(6) - cancel(6) $
+
```

Comment:
no need for the empty lines between block math in tests

Reply:
Good catch.